designer/server/src/common/templates/partials/navigation.njk

```diff
@@ -4,7 +4,7 @@
 {{ appServiceHeader({
   organisationName: "Defra",
   productName: "Forms Designer",
-  navigationItems: navigation if isAuthenticated and isAuthorized,
+  navigationItems: navigation if isAuthenticated and isAuthorized and isFormsUser,
```

Comment:
Thanks @alexluckett

Last of all, what about tests for `isFormsUser: true` with an expired token etc?

We've got a few scenarios where they may no longer be authenticated or their group has changed. Are we still happy flagging `isFormsUser: true` when the others are `false`?

i.e. They might actually find they're an ex-forms user at next sign in

If we're happy with this we'll always need this trio of checks rather than just `isFormsUser: true`

Reply:
Good shout. `isFormsUser` is fine to be true in that scenario though. `isAuthenticated` is the proper way to check for users being logged in, I'd rather stick with this and just use `isFormsUser` as decoration for this use case. Other routes will continue to use the scopes.